engine/install/sles.md

```diff
@@ -44,9 +44,15 @@ The [OpenSUSE `SELinux` repository](https://download.opensuse.org/repositories/s
 must be enabled. This repository is not added by default, and you need to enable
 it for the version of SLES you are running. Run the following commands to add it:
 
+For SLES 15-SP3,
 ```console
```

Comment:
nit: I know we tend to add a newline between text and code-blocks. Perhaps this one should also use a colon (`:`) instead of a comma;

```suggestion
For SLES 15-SP3:

```console
```